Tests/SourceKitLSPTests/TestDiscoveryTests.swift

```diff
@@ -682,10 +682,72 @@ final class TestDiscoveryTests: XCTestCase {
               style: TestStyle.swiftTesting,
               location: Location(uri: uri, range: positions["2️⃣"]..<positions["3️⃣"]),
               children: [],
-              tags: [TestTag(id: "one"), TestTag(id: "two")]
+              tags: [TestTag(id: ".red"), TestTag(id: ".blue")]
             )
           ],
-          tags: [TestTag(id: "Suites")]
+          tags: [TestTag(id: ".green")]
+        )
+      ]
+    )
+  }
+
+  func testSwiftTestingTestWithCustomTags() async throws {
+    let testClient = try await TestSourceKitLSPClient()
+    let uri = DocumentURI.for(.swift)
+
+    let positions = testClient.openDocument(
+      """
+      import Testing
+
+      extension Tag {
+        @Tag static var suite: Self
+        @Tag static var foo: Self
+        @Tag static var bar: Self
+        @Tag static var baz: Self
+
+        struct Nested {
+          @Tag static var foo: Tag
+        }
+      }
+
+      1️⃣@Suite(.tags(.suite))
+      struct MyTests {
+        2️⃣@Test(.tags(.foo, Nested.foo, Testing.Tag.bar, Tag.baz))
```

Comment:
`Nested.foo` is not valid as it's not within the namespace of `Tag`. `.Nested.foo` _is_ valid, however.

Reply:
Oh, ok, so they're _always_ going to have a `.` prefixed. I'll fix it up.